packages/taquito/src/contract/interface.ts

```diff
@@ -238,4 +240,14 @@ export interface ContractProvider extends StorageProvider {
    * @param ProposalsParams Proposals operation parameter
    */
   proposals(params: ProposalsParams): Promise<ProposalsOperation>;
+
+  /**
+   *
+   * @description Update consensus key
+   *
+   * @returns An operation handle with the result from the RPC node
+   *
+   * @param UpdateConsensusKeyParams Proposals operation parameter
```

Comment:
Proposals? Seems to be a copy paste error

Reply:
will update 😄